shared/actions/chat2/index.tsx

```diff
@@ -3706,16 +3714,16 @@ const maybeChangeChatSelection = (action: RouteTreeGen.OnNavChangedPayload, logg
   const p = prev[prev.length - 1]
   const n = next[next.length - 1]
 
-  const wasModal = prev[1]?.routeName !== 'Main'
```

Comment:
the old nav had Main/loggedIn{out}/etc but given the new nav its not just loggedIn{out} at the root